src/aviary/tools/base.py

```diff
@@ -148,14 +146,12 @@ def __str__(self) -> str:
 
 
 class ToolResponseMessage(Message):
-    content: str = Field(
+    content: str | None = Field(
         description=(
             "Response message content, required to be a string by OpenAI/Anthropic."
         ),
     )
-    role: Literal["tool"] = Field(
-        default="tool", description="Matching LiteLLM structure."
-    )
+    role: str = Field(default="tool", description="Matching LiteLLM structure.")
```

Comment:
Can you type ignore this and revert to `Literal`

Let's push this change to later when we know more